website/docs/docs/core/connect-data-platform/teradata-setup.md

```diff
@@ -241,6 +247,14 @@ dbt-teradata 1.8.0 and later versions support unit tests, enabling you to valida
 
 ## Limitations
 
+### Browser Authentication
+When running a dbt job with logmech set to "browser", the initial authentication opens a browser window where you must enter your username and password.<br>
```

Comment:
@tallamohan can you turn these into bullets and remove the <br>. so:

- When running a dbt job with logmech set to "browser", the initial authentication opens a browser window where you must enter your username and password.
- After authentication, this window remains open, requiring you to manually switch back to the dbt console.
- For every subsequent connection, a new browser tab briefly opens, displaying the message "TERADATA BROWSER AUTHENTICATION COMPLETED," and silently reuses the existing session.
- However, the focus stays on the browser window, so you’ll need to manually switch back to the dbt console each time.
- This behavior is the default functionality of the teradatasql driver and cannot be avoided at this time.
- To prevent session expiration and the need to re-enter credentials, ensure the authentication browser window stays open until the job is complete.